src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -170,113 +161,14 @@ public function editAction($featureId, Request $request): Response
         ]);
     }
 
-    /**
-     * @return ControllerConfiguration
-     */
-    public function getControllerConfiguration(): ControllerConfiguration
```

Comment:
All those should stay as well for now at least (everything related to the horizontal page will be cleaned later)